src/components/organisms/AuthenticationModal/RegisterForm/RegisterForm.tsx

```diff
@@ -133,36 +155,46 @@ const RegisterForm: React.FunctionComponent<PropsType> = ({
             placeholder="Password"
             ref={register({
               required: true,
-              // minLength: 8,
-              pattern: /^(?=.*[0-9])(?=.*[a-zA-Z]).{2,}$/,
+              pattern: /^(?=.*[0-9])(?=.*[a-zA-Z]).{6,}$/,
             })}
           />
+
           <span
             className={`input-${
               errors.password && errors.password.type === "pattern"
                 ? "error"
                 : "info"
             }`}
           >
-            Password must contain letters and numbers
+            Password must contain letters, numbers, and be at least 6 characters
+            long
           </span>
+
           {errors.password && errors.password.type === "required" && (
             <span className="input-error">Password is required</span>
           )}
         </div>
+
         {venue.requiresTicketCode && (
           <TicketCodeField register={register} error={errors?.code} />
         )}
+
         {venue.requiresDateOfBirth && (
           <DateOfBirthField register={register} error={errors?.date_of_birth} />
         )}
+
+        {errors.backend && (
+          <span className="input-error">{errors.backend.message}</span>
+        )}
+
```

Comment:
Great that this is right above the button which becomes unclickable; and great that the `onChange` event is used to clear backend errors.